pageserver/src/storage_sync.rs

```diff
@@ -1153,6 +1180,9 @@ enum RemoteDataUpdate<'a> {
         uploaded_data: LayersUpload,
         upload_failed: bool,
     },
+    Download {
```

Comment:
```suggestion
    /// Add the following paths to the set of missing layers as a result of
    /// unexpected missing from remote storage event.
    Download {
```

Or something, I think for me the naming of "Download" was a bit surprising but cannot figure out a better one to suggest either.